fe/src/components/Sidebar/DrawerbarContent.tsx

```diff
@@ -1,20 +1,13 @@
 import { Flex } from '@chakra-ui/react';
-import { IoHome } from 'react-icons/io5';
 import React from 'react';
-import SidebarItem from './SidebarItem';
 import SidebarItemLink from './SidebarItemLink';
 import { NAV_ITEMS } from '../../utils';
 
 export const DrawerbarContent: React.FC = () => (
   <Flex direction="column">
-    {NAV_ITEMS.map(navItem => (
-      <SidebarItem key={navItem.href}>
-        <SidebarItemLink
-          icon={<IoHome size={22} />}
-          title={navItem.label}
-          where={navItem.href}
-        />
-      </SidebarItem>
-    ))}
+    {NAV_ITEMS.map(navItem => {
+      const { iconName, href, label } = navItem;
+      return <SidebarItemLink icon={iconName} title={label} where={href} />;
+    })}
```

Comment:
Found small error for mobile view sidebar, should be easy fix tho. 

![error](https://user-images.githubusercontent.com/48998862/119314396-5ea90080-bc7d-11eb-90cc-681b56acf818.JPG)

Reply:
fixed this

Reply:
i dont know about you, but im still getting this error, trying out something like this fixed it though and also seems simpler for now:

```
        <Box key={href}>
          <SidebarItemLink
            icon={iconName}
            title={label}
            where={href}
          />
        </Box>
```

Reply:
weird when I was testing, it didn't give any errors. will take a look

Reply:
Yep, now its fixed 👌